### ISSUE TITLE
Page Titles for screen reader users for more context

### ISSUE THREAD

**Issue by colabottles** [danielroe/unsight.dev#35]
- Page titles should be descriptive for screen reader users when they switch to a new page or repository.
- This satisfies usability needs to help users find content and orient themselves within it by ensuring that each Web page has a descriptive title.
- This satisfies WCAG guideline [2.4.2 - Page Titled](https://www.w3.org/WAI/WCAG22/Understanding/page-titled.html).

> Titles identify the current location without requiring users to read or interpret page content. When titles appear in site maps or lists of search results, users can more quickly identify the content they need. User agents make the title of the page easily available to the user for identifying the page. For instance, a user agent may display the page title in the window title bar or as the name of the tab containing the page.
>
> In cases where the page is a document or a web application, the name of the document or web application would be sufficient to describe the purpose of the page. Note that it is not required to use the name of the document or web application; other things may also describe the purpose or the topic of the page.
>
> In cases such as Single Page Applications (SPAs), where various distinct pages/views are all nominally served from the same URI and the content of the page is changed dynamically, the title of the page should also be changed dynamically to reflect the content or topic of the current view.

**Comment by colabottles** [comment]
@danielroe I will be happy to take this on.

**Comment by colabottles** [comment]
@danielroe I have figured out and fixed the issue I had messaged you about earlier today so I will be filing a PR soon as possible. Thanks!

**Comment by danielroe** [comment]
resolved in https://github.com/danielroe/unsight.dev/pull/73
